File: introductory_problems/missing_number.cpp
#include <iostream>
#include <iomanip>
#include <math.h>
#include <algorithm>
#include <vector>

using namespace std;

#define DBG(x) cout << "[" << #x << "]: " << x << endl
#define F(x) std::fixed <<std::setprecision(1)<<(x)

int main(int argc, char const *argv[])
{
    long int n, sum = 0, sum_gauss = 0, aux;
    cin >> n;
    for(int i = 0; i < n-1; i++){
        cin >> aux;
        sum += aux;
    }
    sum_gauss = ((1 + n) * n) / 2; 
    aux = sum_gauss - sum;
    cout << aux << '\n';
    return 0;
}

File: introductory_problems/missing_number.py
"""
from re import X


a = 842572599 
b = 577431753
x = 2
y = 1

while(a > 0 and b > 0):
    print(a, b) 
    a -= x
    b -= y 
    aux = x 
    x = y 
    y = aux

print(a, b)
"""

a = 842572599 // 2
a1 = 421286299
b = 577431753 - a1
b1 = 156145454 // 2
b2 = 78072727 * 2
b3 = 156145454 + 156145454
a2 = 78072727 + 1

print()

